server/test/routes/auth.test.ts

```diff
@@ -0,0 +1,101 @@
+// SPDX-FileCopyrightText: 2023 The Aalto Grades Developers
+//
+// SPDX-License-Identifier: MIT
+
+import { app } from '../../src/app';
+import supertest, { SuperAgentTest } from 'supertest';
+import { UserRole } from '../../src/controllers/auth';
+
+const request: supertest.SuperTest<supertest.Test> = supertest(app);
+
+describe('Test login route', () => {
```

Comment:
In our other tests we've used descriptions in the format of `Test METHOD /v1/some/route`, the test descriptions here should follow the same convention.